Resources/Textures/Interface/Misc/health_icons.rsi/meta.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "size": {
+        "x": 8,
+        "y": 8
+    },
+    "license": "CC-BY-3.0",
+    "copyright": "https://tgstation13.org/wiki/File:No_virus_Detected.png (yoinked from TG wiki)",
```

Comment:
TG is BY-SA not BY, also this should be from the repo instead, the wiki also doesn't have a license.